epicbox-hyperskill/go/Dockerfile

```diff
@@ -12,8 +12,6 @@ RUN apt-get update && apt-get install -y \
     gcc \
     linux-headers-amd64 \
     unzip \
-    && python3 -m venv /venv \
-    && . /venv/bin/activate \
```

Comment:
A virtual environment for python should **always** be used. Otherwise, by installing libraries we can break system applications of the operating system

You can try adding `. /venv/bin/activate` in `checker/check.sh`

Reply:
@meanmail we added 
```
&& python3 -m venv /venv \
    && . /venv/bin/activate \
```
in the `golang:1.21.6-alpine` because the secyrity settings in the  linux alpine distro doesn't let me install any Python libraries. Right now we uses Debian and testing in project works correctly without  virtual environment

Reply:
Ok I understand you. You can do without venv if you make sure that installing dependencies does not break the work of other applications